src/lib/mongoBackend/mongoSubCache.cpp

```diff
@@ -0,0 +1,1204 @@
+/*
```

Comment:
Yes, I had the same thought. That's why I put "v2" of the cache in mongoBackend.
At least, it has its own file ...